src/test/java/io/managed/services/test/billing/BillingMetricsTest.java

```diff
@@ -76,8 +76,8 @@ public class BillingMetricsTest extends TestBase {
 
     // TODO metrics will soon be renamed to match new naming convention, more at MGDSTRM-7080.
     private static final String METRIC_STORAGE = "kafka_broker_quota_totalstorageusedbytes";
-    private static final String METRIC_TRAFFIC_IN = "haproxy_server_bytes_in_total";
-    private static final String METRIC_TRAFFIC_OUT = "haproxy_server_bytes_out_total";
+    private static final String METRIC_TRAFFIC_IN = "haproxy_backend_bytes_in_total";
```

Comment:
if the tests need to make observations of the _underlying_ metric, you need need to use  `kafka_instance_incoming_byte_total` instead.  The namespace label is called `namespace` rather than `exported_namespace`

 https://github.com/bf2fc6cc711aee1a0c2a/observability-resources-mk/blob/main/resources/prometheus/kafka-recording-rules.yaml#L39

There's also `kafka_instance_outgoiing_byte_total`